modules/stochastic_tools/include/surrogates/LibtorchSimpleNNSurrogate.h

```diff
@@ -0,0 +1,40 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#ifdef TORCH_ENABLED
+#include <torch/torch.h>
+#include "LibtorchSimpleNeuralNet.h"
+#endif
+
+#include "SurrogateModel.h"
+
+class LibtorchSimpleNNSurrogate : public SurrogateModel
+{
+public:
+  static InputParameters validParams();
+  LibtorchSimpleNNSurrogate(const InputParameters & parameters);
+
+  using SurrogateModel::evaluate;
+  virtual Real evaluate(const std::vector<Real> & x) const override;
+
+protected:
+  /// Number of hidden layers in the neural net
+  const unsigned int & _no_hidden_layers;
```

Comment:
I always read "no hidden  layers "  :-)

Reply:
change "no" to "num"?

Reply:
Yeah I usually use no = "number of", but I'll change it to num in the next commit!

Reply:
Fixed!